docs/quality-assurance-test-cases.md

```diff
@@ -172,3 +172,13 @@ Only admin users have access to this.
 | Revoke self the admin role when self is not the only admin | Success |
 | Revoking an admin user, when the current user is not an admin | Fail |
 
+## Tasks
+
+### Create
+**Service:** POST /mentorship_relation/{relation_id}/task
+
+| Test Case       | Outcome |
+| ------------- | ------------- |
+|Create a task if there is a valid relation exist between user and mentor and relation state is accepted | Success |
+| Create a task if there is no relation between user and mentor | Fail |
```

Comment:
I am not sure if anyone could even test this, can you be more specific? I think what you want to say here is something like:
create a task when logged user is not involved in the relation